spec.html

```diff
@@ -648,28 +651,46 @@ <h1><ins>ExecuteAsyncModule ( _module_ )</ins></h1>
       </emu-alg>
     </emu-clause>
 
+    <emu-clause id="sec-gather-async-parent-completions" aoid="GatherAsyncParentCompletions">
+      <h1><ins>GatherAsyncParentCompletions ( _module_, _execList_ )</ins></h1>
+      <emu-alg>
+        1. Assert: _module_.[[Status]] is ~evaluated~.
+        1. For each Module _m_ of _module_.[[AsyncParentModules]], do
+          1. If _execList_ does not contain _m_ and _m_.[[CycleRoot]].[[EvaluationError]] is ~empty~, then
+            1. Assert: _m_.[[EvaluationError]] is ~empty~.
+            1. Assert: _m_.[[AsyncEvaluating]] is *true*.
+            1. Assert: 0 < _m_.[[PendingAsyncDependencies]].
```

Comment:
let’s avoid a yoda condition :-)
```suggestion
            1. Assert: _m_.[[PendingAsyncDependencies]] > 0.
```